docs/java/features/odata/generate-client.mdx

```diff
@@ -97,94 +97,74 @@ Furthermore, some common IDEs (e.g. IntelliJ, Eclipse) require plugins to recogn
 
 ### Using the OData Generator Maven Plugin ###
 
+The maven plugin is most useful if you would like to use the generated code within a maven project.
+Use it by adding it to your `application/pom.xml` file under the `<plugin>` section:
+
 <Tabs groupId="odataVersion" defaultValue="v4" values={[ { label: 'OData v2', value: 'v2', }, { label: 'OData v4', value: 'v4', }]}>
 <TabItem value="v4">
 
-1. Update your `application/pom.xml` file by adding the generator plugin under the `<plugin>` section.
-
-    ```xml
-    <plugin>
-        <groupId>com.sap.cloud.sdk.datamodel</groupId>
-        <artifactId>odata-v4-generator-maven-plugin</artifactId>
-        <!-- Please use the latest version here-->
-        <version>3.XX.X</version>
-        <executions>
-            <execution>
-                <id>generate-consumption</id>
-                <phase>generate-sources</phase>
-                <goals>
-                    <goal>generate</goal>
-                </goals>
-                <configuration>
-                    <inputDirectory>${project.basedir}/edmx</inputDirectory>
-                    <outputDirectory>${project.build.directory}/vdm</outputDirectory>
-                    <deleteOutputDirectory>true</deleteOutputDirectory>
-                    <packageName>com.mycompany.vdm</packageName>
-                    <defaultBasePath>odata/v4/</defaultBasePath>
-                </configuration>
-            </execution>
-        </executions>
-    </plugin>
-    ```
+```xml
+<plugin>
+    <groupId>com.sap.cloud.sdk.datamodel</groupId>
+    <artifactId>odata-v4-generator-maven-plugin</artifactId>
+    <!-- Please use the latest version here-->
+    <version>3.XX.X</version>
+    <executions>
+        <execution>
+            <id>generate-consumption</id>
+            <phase>generate-sources</phase>
+            <goals>
+                <goal>generate</goal>
+            </goals>
+            <configuration>
+                <inputDirectory>${project.basedir}/edmx</inputDirectory>
+                <outputDirectory>${project.build.directory}/vdm</outputDirectory>
+                <deleteOutputDirectory>true</deleteOutputDirectory>
+                <packageName>com.mycompany.vdm</packageName>
+                <defaultBasePath>odata/v4/</defaultBasePath>
```

Comment:
```suggestion
                <defaultBasePath>odata/v2/</defaultBasePath>
```

Reply:
The suggestion was good but on the wrong section ;)

Reply:
How do you find the text that corresponds to a particular tab when you have multiple tabs? I find it difficult to find in a markdown file.

Reply:
Look for the `<TabItem value="v2">` above the code. Everything between that and `</TabItem>` belongs to the `v2` tab.

Reply:
Ah okay, missed that. Thanks for pointing it out.